README.md

```diff
@@ -51,7 +51,11 @@ The above assumes you are running on OS X. Put the platform in you are running o
 KIBANA_HOST=http://localhost:5601 KIBANA_USERNAME=elastic KIBANA_PASSWORD=changeme ELASTICSEARCH_HOST=http://localhost:9200 ELASTICSEARCH_USERNAME=elastic ELASTICSEARCH_PASSWORD=changeme KIBANA_FLEET_SETUP=1 FLEET_SERVER_ENABLE=1 sudo ./elastic-agent container
 ```
 
-This will start up Elastic Agent with fleet-server and directly enroll it. In addition Fleet is setup inside of Kibana.
+This will start up Elastic Agent with fleet-server and directly enroll it. In addition Fleet is setup inside of Kibana. In case the setup is done already in Kibana manually, the following command can be used:
+
+```
+sudo ./elastic-agent enroll --fleet-server=http://elastic:changeme@localhost:9200 --fleet-server-policy={fleet-server-policy-id} --enrollment-token={policy-enrollment-token}
```

Comment:
@blakerouse Few questions here:

* For the Elasticsearch host, should we make it possible to not have to pass it all in one string?
* enrollment token: Why is the enrollment token needed as I provided already the fleet-server-policy? Does the Agent itself still need an enrollment token if it runs fleet-server?

Reply:
I prefer it all in one string for the elasticsearch host especially with username/password, I think it is cleaner and easier to understand. That might not hold up with service tokens, but I think its a cleaner syntax.

As for the enrollment token, as we discussed this could be removed.